backend/build.gradle

```
@@ -38,8 +38,11 @@ dependencies {
     asciidoctor 'org.springframework.restdocs:spring-restdocs-asciidoctor'
 
     // lombok
-    annotationProcessor 'org.projectlombok:lombok'
-    testAnnotationProcessor 'org.projectlombok:lombok'
+    annotationProcessor('org.projectlombok:lombok')
+    testAnnotationProcessor('org.projectlombok:lombok')
```

```suggestion
    annotationProcessor 'org.projectlombok:lombok'
    testAnnotationProcessor 'org.projectlombok:lombok'
```
B04 브랜치에서 다른 의존성 관련 코드들과 통일성을 위해 괄호를 없애는 작업을 했는데, Conflict를 해결하는 과정에서 다시 붙은 것 같아서 남깁니다 !

---

감사합니다~~!! 적용할게유👍